README.md

```diff
@@ -39,6 +39,10 @@ Include the CSS and JavaScript
 
 ## Installation
 
+To install GOV.UK Frontend NPM package in your application, need to create a [package.json](https://docs.npmjs.com/files/package.json) file in your application (if you haven't done so already) and add a package to it's dependencies section.
```

Comment:
Should we be telling people to add packages to their package.json and then below telling them to run a command that would do this for them?

Reply:
duh :( fixed